pkg/proxy/ipvs/proxier.go

```diff
@@ -168,6 +168,9 @@ const sysctlForward = "net/ipv4/ip_forward"
 const sysctlArpIgnore = "net/ipv4/conf/all/arp_ignore"
 const sysctlArpAnnounce = "net/ipv4/conf/all/arp_announce"
 
+const ipvsServiceSchedulerAnnotation = "proxy.ipvs.alpha.k8s.io/scheduler"
```

Comment:
Without reviewing the whole PR yet (sorry!) I will say this:

* annotations and labels are "...kubernetes.io/..." rather than k8s.io
* embedding "alpha" in the name just punishes people who use it

kube-proxy.kubernetes.io/ipvs-scheduler is better